browser/ts/index.ts

```diff
@@ -701,14 +705,14 @@ function wrapperContractEventsToContractEvents(wrapperContractEvents: WrapperCon
                 break;
             case ContractEventKind.ERC1155TransferBatchEvent:
                 const erc1155TransferBatchEvent = rawParameters as WrapperERC1155TransferBatchEvent;
-                const ids: BigNumber[] = []
```

Comment:
@fabioberger This formatting change was made automatically by TSLint/Prettier. Can you double check that both of those are running automatically in your editor setup? I'll also add the corresponding checks to CI in a separate PR.

Reply:
FYI opened the corresponding issue here: https://github.com/0xProject/0x-mesh/issues/460